demo/c/cheetah_demo_file.c

```diff
@@ -172,19 +178,53 @@ int picovoice_main(int argc, char **argv) {
         exit(1);
     }
 
-    void (*pv_free_func)(void *) = load_symbol(dl_handle, "pv_free");
-    if (!pv_free_func) {
-        print_dl_error("failed to load `pv_free`");
+    pv_status_t (*pv_cheetah_transcript_delete_func)(char *) =
+            load_symbol(dl_handle, "pv_cheetah_transcript_delete");
+    if (!pv_cheetah_transcript_delete_func) {
+        print_dl_error("failed to load `pv_cheetah_transcript_delete`");
+        exit(1);
+    }
+
+    pv_status_t (*pv_get_error_stack_func)(char ***, int32_t *) = load_symbol(dl_handle, "pv_get_error_stack");
+    if (!pv_get_error_stack_func) {
+        print_dl_error("failed to load 'pv_get_error_stack_func'");
+        exit(1);
+    }
+
+    void (*pv_free_error_stack_func)(char **) = load_symbol(dl_handle, "pv_free_error_stack");
+    if (!pv_free_error_stack_func) {
+        print_dl_error("failed to load 'pv_free_error_stack_func'");
         exit(1);
     }
 
     struct timeval before;
     gettimeofday(&before, NULL);
 
+    char **message_stack = NULL;
+    int32_t message_stack_depth = 0;
+    pv_status_t error_status = PV_STATUS_RUNTIME_ERROR;
+
     pv_cheetah_t *cheetah = NULL;
     pv_status_t status = pv_cheetah_init_func(access_key, model_path, 0.f, enable_automatic_punctuation, &cheetah);
     if (status != PV_STATUS_SUCCESS) {
         fprintf(stderr, "failed to init with `%s`.\n", pv_status_to_string_func(status));
+        error_status = pv_get_error_stack_func(&message_stack, &message_stack_depth);
+        if (error_status != PV_STATUS_SUCCESS) {
+            fprintf(
+                    stderr,
+                    ".\nUnable to get Cheetah error state with '%s'.\n",
+                    pv_status_to_string_func(error_status));
+            exit(1);
+        }
+
+        if (message_stack_depth > 0) {
+            fprintf(stderr, ":\n");
+            print_error_message(message_stack, message_stack_depth);
+        } else {
+            fprintf(stderr, ".\n");
+        }
+
+        pv_free_error_stack_func(message_stack);
```

Comment:
the prints when you get a pv error are pretty weird. Irregular spacing and periods. Check out the slight changes I made in the leopard C demo to improve the look of the output.